README.md

```diff
@@ -40,10 +40,15 @@ Here are the important ones you're likely to use:
 - `npm start`: starts the app for development
 - `npm run build`: will build the documentation site and generate all static files in "build". After `build` you can
   run `npm run serve` to test your build locally
-- `lint`: runs a format check and if no error is found, lints code and markdown files in the project.
+- `npm run lint`: runs a format check and if no error is found, lints code and markdown files in the project.
   - the linting script does not mutate your code. See [Linting and code formatting](#linting-and-code-formatting) for
     more info.
-- `reformat`: runs the code formatter (prettier) as well as the markdown linter in fix mode. This will mutate you code.
+- `npm run reformat`: runs the code formatter (prettier) as well as the markdown linter in fix mode. This will mutate
+- `npm run markdown-link-check`: checks if there are any broken links. Note: This requires internet connection, as it
+  does check external links, too!
+  - Run `npm run markdown-link-check -- -o` to only check for internal links (offline mode).
+  - Run `npm run markdown-link-check -- -q` to only log failures.
+  - Run `npm run markdown-link-check -- -f /path/your-file.md` to run checks for only one file.
 
 ℹ️ We are using a custom hook path for enabling pre-commit hooks. This path is set in the local git config when
```

Comment:
I had to run npm install before i could run lint checker so should we mention that at the start ?

Reply:
Good point, added it like this: https://github.com/Aiven-Open/klaw-docs/pull/87/commits/6858954a32069e7da7df6655557efef19c02fb27